boards/imix/src/aes_ccm_test.rs

```diff
@@ -1,3 +1,16 @@
+//! To run this test, include the code ```
```

Comment:
Minor thing: Fenced code blocks need to start on their own line (not sure if this is a general markdown rule or a rust+markdown rule), but currently this looks like:

<img width="780" alt="image" src="https://user-images.githubusercontent.com/339422/48878843-19f73280-edbe-11e8-98df-67a8cb14aaa5.png">

https://deploy-preview-1219--docs-tockosorg.netlify.com/imix/aes_ccm_test/index.html

Reply:
Fixed.